test/dynamo/test_autograd_function.py

```diff
@@ -281,7 +281,7 @@ def test_stride_in_bwd(self):
         x = torch.randn(2, 2, dtype=torch.double, requires_grad=True)
         with self.assertRaisesRegex(
             torch._dynamo.exc.Unsupported,
-            ".*HigherOrderOperator body's output must consist of tensors only",
+            "Illegal getattr invocation stride in strict mod",
```

Comment:
This was accidentally changed by https://github.com/pytorch/pytorch/commit/28bb31e4a588f3fa198e927439fe09c871201aab#diff-6604f3e9a4e51c5e65634ca48b897932ac03b1837efb27a3f3b3948f99567c6bL278

Reply:
Is it possible to add a stricter test so we don't regress on this in the future?

Reply:
Do you mean a stricter test that not relying on exception message? I think we can. Let me add it in a follow up PR.

Reply:
Yes

Reply:
https://github.com/pytorch/pytorch/pull/119237